strutil/strutil.go

```diff
@@ -127,3 +127,14 @@ func SortedListContains(list []string, str string) bool {
 	}
 	return list[i] == str
 }
+
+// IsPrintableASCII returns whether all characters in the given string are in the
+// printable (non-whitespace) ASCII range (that's everything 0x21-0x7e inclusive).
+func IsPrintableASCII(str string) bool {
```

Comment:
I just noticed the `regexp` package calls this range “graphical”, so `IsGraphicalASCII` would be a better name (because “printable” typically includes ` ` and, depending on context, `\t\n\v\f\r`).